docs/content/docs/libs/state_processor_api.md

```diff
@@ -515,6 +515,28 @@ Before you interrogate state using the table API, make sure to review our [Flink
 
 IMPORTANT NOTE: State Table API only supports keyed state.
 
+### Metadata
+
+The SQL connector allows users to read metadata of savepoints in the following way:
+```SQL
+LOAD MODULE state;
+SELECT * FROM savepoint_metadata('/root/dir/of/checkpoint-data/chk-1');
```

Comment:
Thr flip talks of checkpoints and savepoint state being accessible, but the function is savepoint_metadata. Can we function change the name so it applies to both savepoints and checkpoints - maybe `state_metadata`?

Reply:
> The flip talks of checkpoints and savepoint state, can we use this funciton to access checkpoint state as well? 

Yes. We can't name things in both ways. Initially we've agreed to call the whole state processor API things as `savepoint`... and the only exception here is the module because it's a reserved keyword in Flink SQL. If we go on this road, then the whole state processor API including internals must be renamed which I don't support.